bin/install_nodejs

```diff
@@ -23,6 +23,8 @@ if needs_resolution "$version"; then
 fi
 
 cd /tmp
+# Key Download
+gpg --keyserver pool.sks-keyservers.net --recv-keys 77984A986EBC2AA786BC0F66B01FBB92821C587A
```

Comment:
Any reason we wouldn't import the [full set of release keys](https://github.com/nodejs/node#release-team) here?

Reply:
Nope, let me update that ;)